src/routing/Router.tsx

```diff
@@ -7,6 +8,10 @@ export const Router = () => {
     <Routes>
       <Route path='/' element={<DefaultLayout />}>
         <Route index element={<Home />} />
+        <Route
+          path='compound-component'
+          element={<CompoundComponentPage></CompoundComponentPage>}
```

Comment:
Acá no hace falta usar tag de apertura y cierre, es suficiente solo con una
```suggestion
          element={<CompoundComponentPage />}
```